src/python/espressomd/_system.pyx

```diff
@@ -33,10 +33,11 @@ from cellsystem import CellSystem
 
 import sys
 
-setable_properties=["box_l","max_num_cells","min_num_cells",
-                    "node_grid","npt_piston","npt_p_diff",
-                    "periodicity","skin","time",
-                    "time_step","timings"]
+__seed=-10
```

Comment:
yes it is needed, since __seed otherwise is created in __set___() but then the property seed is not recognized as such by python